packages/es-components/package.json

```diff
@@ -74,12 +76,15 @@
     "react-styleguidist": "^9.1.11",
     "react-test-renderer": "^16.8.6",
     "react-testing-library": "^6.1.2",
-    "rollup": "^1.15.6",
+    "rollup": "^2.8.2",
     "rollup-plugin-babel": "^4.3.2",
     "rollup-plugin-commonjs": "^10.0.1",
+    "rollup-plugin-generate-html-template": "^1.6.1",
+    "rollup-plugin-node-builtins": "^2.1.2",
     "rollup-plugin-node-resolve": "^5.2.0",
     "rollup-plugin-uglify": "^6.0.4",
     "styled-components": "^4.3.2",
+    "svg-inline-loader": "^0.8.2",
```

Comment:
I don't see this or the `file-loader` above being used anywhere. Are they leftover from something you were trying? Or is there a non-obvious usage of them somewhere?

Reply:
They are left over. I can remove them.